### PR TITLE
Update install.sh to set "experimental" to true if needed

### PR DESCRIPTION
**Pull request by mlaventure**
Signed-off-by: Kenfe-Mickael Laventure mickael.laventure@gmail.com
## 

This is a very conservative approach. We only update `/etc/docker/daemon.json` if it doesn't exist yet.

### CODE REVIEW

File: hack/install.sh [hunk 1]
```diff
@@ -23,6 +23,28 @@ set -e
 #     aws s3 cp --acl public-read hack/install.sh s3://get.docker.com/index
 #
 
+function update_daemon_config() {
+	[ $? = 0 ] || return
+
+	config="/etc/docker/daemon.json"
+	config="./daemon.json"
```
Comment (vdemeester) [inline hack/install.sh]: Isn't this one not supposed to be here ? :stuck_out_tongue_closed_eyes:

Comment (mlaventure) [inline hack/install.sh]: Nop, I was debugging :sweat_smile:

**Comment by mlaventure** [comment]
Actually, this may need to be done in the `.deb`/`rpm` since I think they start the daemon as soon as the install is done.

cc @andrewhsu @tianon

**Comment by vieux** [comment]
@andrewhsu @tianon please take a look :)

**Comment by tianon** [comment]
I'm personally -1 and would rather we document this well (and perhaps have the script link directly to the relevant section of the documentation for how to enable this).

Is this one of the configuration items supported for live daemon configuration reload, or does this require a full daemon restart when changed?

**Comment by vieux** [comment]
If I'm not mistaken, for now it requires a full daemon restart

**Comment by mlaventure** [comment]
It requires a full restart indeed.

We could just add a message at the end stating that the option needs to be
added either to the service file or the daemon configuration file.

**Comment by andrewhsu** [comment]
@mlaventure yes, `daemon.json` should probably be a change in the postinstall step of deb and rpm packages that are built when `DOCKER_EXPERIMENTAL=1` is set instead of a change in `install.sh`.

**Comment by mlaventure** [comment]
Alright, closing this then.

**Pull request closed without merging**